### PR TITLE
Game teams factory nico #8

### PR DESCRIPTION
**Pull request by NicoShanstrom**
Created files for game_teams and game_teams_factory in lib (was not able to make any methods in them yet)
created files in spec for game_teams_spec and game_teams_factory_spec and wrote tests.

### CODE REVIEW

File: spec/game_teams_factory_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        end
+
+        it 'creates objects with the necessary attributes' do
+
+            expect(@game_teams_factory.create_game_teams.first.game_id).to eq(2012030221)
+            expect(@game_teams_factory.create_game_teams.first.team_id).to eq(3)
+            expect(@game_teams_factory.create_game_teams.first.hoa).to eq("away")
+            expect(@game_teams_factory.create_game_teams.first.result).to eq("LOSS")
+            expect(@game_teams_factory.create_game_teams.first.settled_in).to eq("OT")
```
Comment (JackCSweeney) [inline spec/game_teams_factory_spec.rb:39]: This looks great! Good catch on setting the tests to grab the integers for the numbers that are coming in and strings for everything that isn't set that way!

File: spec/game_teams_spec.rb [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        end
+
+        it 'returns pim aka penulty minutes for a team' do
+
+            expect(@game_teams.pim).to eq(8)
+        
+        end
+
+        it 'returns power play opportunities for a team' do
+            #should i keep the method name consistent with the hash key, or change the hash key
```
Comment (JackCSweeney) [inline spec/game_teams_spec.rb]: Good question. Ruby convention would tell us to make it into something different than what we're seeing in our csv file if it doesn't fit that snake_case convention that we should be using. So you should change this one to power_play_opportunities or power_play_opps like you mentioned in the comment.

**Review by JackCSweeney — CHANGES_REQUESTED** [review]

File: spec/game_teams_factory_spec.rb (hunk 1 above)
Comment (NicoShanstrom) [reply spec/game_teams_factory_spec.rb:39]: nice

File: spec/game_teams_spec.rb (hunk 2 above)
Comment (NicoShanstrom) [reply spec/game_teams_spec.rb]: modified method names to be snake case power_play_opps, power_play_goals, and faceoff_win_percent

File: spec/game_teams_spec.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+        it 'returns power play opportunities for a team' do
+ 
+            expect(@game_teams.power_play_opps).to eq(3)
+
+        end
+
+        it 'returns power play goals for a team' do
+            
+            expect(@game_teams.power_play_goals).to eq(0)
```
Comment (JackCSweeney) [inline spec/game_teams_spec.rb:94]: Looks great! Thank you for making that change!

**Review by JackCSweeney — APPROVED** [review]

**Pull request merged**